integrations/utils.py

```diff
@@ -53,3 +54,22 @@ def parse_score(output, metric, digits=4):
             score = float(line.split()[-1])
             return round(score, digits)
     return None
+
+
+# Function for parsing the output from pyserini.eval.evaluate_dpr_retrieval
+def parse_score_qa(output, metric, digits=4):
+    for line in output.split('\n'):
+        if metric in line:
+            score = float(line.split()[-1])
+            return round(score, digits)
+    return None
+
+
+# Function for parsing the output from MS MARCO eval scripts
+# (Currently, parses the same way as parse_score_qa above, but keeping separate in case they diverge in the future.)
+def parse_score_msmarco(output, metric, digits=4):
+    for line in output.split('\n'):
+        if metric in line:
+            score = float(line.split()[-1])
+            return round(score, digits)
```

Comment:
nit: why do we need rounding here?

Reply:
Due to floating point precision issues, Python might parse string `0.2000` and internally it becomes 0.19999999. We make sure this doesn't happen...

Reply:
Got it, thanks!